Donation-platform/src/main/java/com/google/DbCalls.java

```diff
@@ -0,0 +1,135 @@
+package com.google.servlets;
+
+import com.google.model.Charity;
+import com.google.model.Tag;
+import com.google.model.Users; 
+import java.util.HashMap;
+import java.util.List;
+import java.util.ArrayList;
+import java.util.Map;
+import java.util.Collection;
+import java.util.Collections;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory;
+
+
+/**
+ * Class handling the calls to the datastore database.
+ */
+public final class DbCalls {
+
+  private DatastoreService datastore;
+  // Function initializes the datastore variable when called.
+  public void initializeDatastore(){
```

Comment:
instead of this function, you can initialize in constructor and take DataStore as a parameter. Will also help in testing,

Reply:
resolved